Host/Mdis_Test/Mdis_Test.sh

```diff
@@ -414,11 +414,9 @@ if [ "${RUN_INSTANTLY}" == "1" ]; then
 else
     grub_set_os "0"
     for ExpectedOs in "${GrubOses[@]}"; do
-        ssh-keygen -R "${MenPcIpAddr}"
         # Check if devices are available
-        if ! ping -i 5 -c 4  "${MenPcIpAddr}"
-        then
-            echo "${MenPcIpAddr} is not responding"
+        if ! sshpass -p "${MenPcPassword}" ssh mdis-setup${TEST_SETUP} "echo"; then
```

Comment:
As this line appears several times, could be interesting to move this line to a function.